public/app/features/alerting/unified/components/rule-editor/AnnotationHeaderField.tsx

```diff
@@ -38,8 +38,10 @@ const AnnotationHeaderField = ({
               switch (annotationField.key) {
                 case Annotation.dashboardUID:
                   label = 'Dashboard and panel';
+                  break;
                 case Annotation.panelID:
                   label = '';
+                  break;
                 default:
```

Comment:
this was a 🐛

Reply:
nice catch!

Reply:
I wonder why `dashboardUID` has `Dashboard and panel` label while `panelID` is empty?